packages/app/src/components/pages/Home/useSort.js

```diff
@@ -55,11 +56,14 @@ const useSort = () => {
     return sortableNoms
   }, [NominationsData, sortConfig])
 
-  const requestSort = (key) => {
-    let direction = 'ascending'
-
-    if (sortConfig && sortConfig.key === key && sortConfig.direction === 'ascending') {
-      direction = 'descending'
+  const requestSort = (key, doDefault) => {
+    console.log(doDefault)
+    let direction= SORT_DIRECTION.UP
+    if (!doDefault && sortConfig.key === key && sortConfig.direction === SORT_DIRECTION.UP) {
+      direction = SORT_DIRECTION.DOWN
+    }
+    if (doDefault) {
+      direction= SORT_DIRECTION.UP
```

Comment:
Currently `direction` is being assigned in three places. Can you think of a way so it only has to be assigned in two places?

Reply:
Absolutely, we will get that taken care of this evening.

Reply:
This has been corrected.